src/test/scala/org/broadinstitute/dsde/rawls/expressions/SlickSimpleExpressionParserTest.scala

```diff
@@ -0,0 +1,230 @@
+package org.broadinstitute.dsde.rawls.expressions
+
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.broadinstitute.dsde.rawls.dataaccess.slick.{TestDriverComponent, TestDriverComponentWithFlatSpecAndMatchers}
+import org.broadinstitute.dsde.rawls.dataaccess.{SlickWorkspaceContext, RawlsTransaction, WorkspaceContext}
+import org.broadinstitute.dsde.rawls.graph.OrientDbTestFixture
+import org.broadinstitute.dsde.rawls.model._
+import org.scalatest.FunSuite
+import org.scalatest.TryValues
+
+import scala.collection.mutable.Seq
+import scala.util.{Failure, Success}
+
+/**
+ * Created by abaumann on 5/21/15.
+ */
+class SlickSimpleExpressionParserTest extends FunSuite with TestDriverComponent {
+  def withTestWorkspace(testCode: (SlickWorkspaceContext) => Any): Unit = {
+    withDefaultTestDatabase {
+      withWorkspaceContext(testData.workspace) { workspaceContext =>
+        testCode(workspaceContext)
+      }
+    }
+  }
+
+  test("simple attribute expression") {
+    withTestWorkspace { workspaceContext =>
+      val evaluator = expressionEvaluator
+      assertResult(Seq(AttributeString("normal"))) {
+        runAndWait(evaluator.evalFinalAttribute(workspaceContext, "Sample", "sample1", "this.type").get)
+      }
+
+      assertResult(Seq(AttributeString("normal"), AttributeString("tumor"), AttributeString("tumor"))) {
+        runAndWait(evaluator.evalFinalAttribute(workspaceContext, "SampleSet", "sset1", "this.samples.type").get)
+      }
+
+      assertResult(Seq()) {
+        runAndWait(evaluator.evalFinalAttribute(workspaceContext, "SampleSet", "sset_empty", "this.samples").get)
+      }
+
+      assertResult(Seq(AttributeString("a"), AttributeString("b"))) {
+        runAndWait(evaluator.evalFinalAttribute(workspaceContext, "Sample", "sample1", "this.thingies").get)
+      }
+    }
+  }
+
+  test("string literals") {
```

Comment:
I guess I don't understand the literals tests.  Is this simply demonstrating that only "this." and "workspace." expressions are valid?